### PR TITLE
modify version number 0.5.1 to 0.6.0

### PR DESCRIPTION
**Pull request by lacrimosaprinz**

### CODE REVIEW

File: docs/MatrixOne/Develop/export-data/select-into-outfile.md [hunk 1]
```diff
@@ -37,15 +37,14 @@ The `SELECT ... INTO OUTFILE` has the following properties −
...
-- Use MySQL client to [connect to MatrixOne](https://docs.matrixorigin.io/0.5.1/MatrixOne/Get-Started/connect-to-matrixone-server/).
+- Make sure you have already [installed and launched MatrixOne](../../Get-Started/install-standalone-matrixone.md).
+- Use MySQL client to [connect to MatrixOne](../../Get-Started/connect-to-matrixone-server.md).
 
 !!! note
     If you install MatrixOne by `docker`, the directory is inside the docker image by default. To work with local directory, you need to bind a local directory to the container. In the following example, the local file system path `/tmp/docker_export_demo/store` is binded to the MatrixOne docker image, with a mapping to the `/store` path. For more information, see [Docker Mount Volume tutorial](https://www.freecodecamp.org/news/docker-mount-volume-guide-how-to-mount-a-local-directory/).
 
 ```
-docker run -d -p 6001:6001 -v ~/tmp/docker_export_demo/store:/store:rw --name matrixone matrixorigin/matrixone:0.5.1
+docker run -d -p 6001:6001 -v ~/tmp/docker_export_demo/store:/store:rw --name matrixone matrixorigin/matrixone:0.6.0
```
Comment (dengn) [inline docs/MatrixOne/Develop/export-data/select-into-outfile.md:47]: this docker mount directory needs to be updated.

File: docs/MatrixOne/Reference/SQL-Reference/Explain/explain.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ where option can be one of:
     FORMAT { TEXT | JSON }
 ```
 
-For 0.5.1, only `TEXT` format is supported.
+For 0.6.0, only `TEXT` format is supported.
```
Comment (dengn) [inline docs/MatrixOne/Reference/SQL-Reference/Explain/explain.md:16]: Please verify if these limitations are still the same. @domingozhang

**Review by dengn — CHANGES_REQUESTED** [review]

File: docs/MatrixOne/Reference/SQL-Reference/Explain/explain.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ Specifies whether the selected option should be turned on or off. You can write
 
 * STETEMENT
 
-MatrixOne supports any `SELECT`, `UPDATE`, `DELETE` statement execution plan. For `INSERT` statement, only `INSERT INTO..SELECT` is supported in 0.5.1 version. `INSERT INTO...VALUES` is not supported yet.
+MatrixOne supports any `SELECT`, `UPDATE`, `DELETE` statement execution plan. For `INSERT` statement, only `INSERT INTO..SELECT` is supported in 0.6.0 version. `INSERT INTO...VALUES` is not supported yet.
```
[the remark this answers is not in the document]
Comment (domingozhang) [reply docs/MatrixOne/Reference/SQL-Reference/Explain/explain.md:46]: insert still not supported

[the remark this answers is not in the document]
Comment (domingozhang) [reply docs/MatrixOne/Reference/SQL-Reference/Explain/explain.md:46]: insert into still not supported

File: docs/MatrixOne/Reference/SQL-Reference/Explain/explain.md (hunk 2 above)
Comment (domingozhang) [reply docs/MatrixOne/Reference/SQL-Reference/Explain/explain.md:16]: json is not supported

File: docs/MatrixOne/Develop/export-data/export-data.md [hunk 4]
```diff
@@ -4,14 +4,13 @@ This document will guide you how to export data in MatrixOne.
...
-
-- Use MySQL client to [connect to MatrixOne](https://docs.matrixorigin.io/0.5.1/MatrixOne/Get-Started/connect-to-matrixone-server/). 
+- Make sure you have already [installed and launched MatrixOne](../../Get-Started/install-standalone-matrixone.md).
+- Use MySQL client to [connect to MatrixOne](../../Get-Started/connect-to-matrixone-server.md).
 
 If you use the `docker` install, please make sure that you have a data directory mounted to the container. For example,
 
 ```
-docker run -d -p 6001:6001 -v ~/tmp/docker_export_demo/store:/store:rw --name matrixone matrixorigin/matrixone:0.5.1
+docker run -d -p 6001:6001 -v ~/tmp/docker_export_demo/store:/store:rw --name matrixone matrixorigin/matrixone:0.6.0
```
Comment (dengn) [inline docs/MatrixOne/Develop/export-data/export-data.md:13]: this docker directory path needs to be updated.

**Review by dengn — CHANGES_REQUESTED** [review]

**Review by dengn — APPROVED** [review]

**Comment by dengn** [comment]
Please resolve conflict @lacrimosaprinz

**Pull request closed without merging**